server/integration/infinispan/src/main/resources/subsystem-templates/infinispan-core.xml

```diff
@@ -29,27 +23,27 @@
             <transport lock-timeout="60000" />
             <global-state />
 
-            <distributed-cache-configuration name="transactional" mode="SYNC" start="EAGER">
+            <distributed-cache-configuration name="transactional" mode="SYNC">
```

Comment:
Yes, this sounds like a good idea. Having a compulsory attribute has always irked me a bit, and SYNC as a default is definitely a good default.